frontend-react/src/styles/_uswds_overrides.scss

```diff
@@ -412,6 +416,21 @@ body {
     h6 {
         @include h6;
     }
+
+    // If header parent has font rule applied, inherit it instead
```

Comment:
Should any of this be scoped to an `Accordian.module.scss` file?

Reply:
No, this applies to more than just accordion and style rules inside of a `component.module.scss` should only be style rules specific to that custom component, not general overrides.